libmamba/src/core/virtual_packages.cpp

```diff
@@ -236,7 +236,52 @@ namespace mamba
             {
                 return "x86_64_v2";
             }
+#elif defined(_MSC_VER) && defined(_M_X64)
+            // We apply boolean masks to the results of the cpuid instruction
+            // to determine if the CPU supports the required instruction sets.
+            // The masks are generated by shifting 1 to the left by the bit
+            // position of the required feature.
+            //
+            // See those sources to understand bitshifts:
+            // https://docs.microsoft.com/en-us/cpp/intrinsics/cpuid-cpuidex?view=msvc-160
+            // https://en.wikipedia.org/wiki/CPUID#EAX=7,_ECX=0:_Extended_Features
```

Comment:
I think a link to https://en.wikipedia.org/wiki/X86-64#Microarchitecture_levels might be worth too (for the version spec of x86_64)